apps/web/components/Embed.tsx

```diff
@@ -979,6 +984,8 @@ const EmbedTypeCodeAndPreviewDialogContent = ({
                     <div>Text Color</div>
                     <div className="w-full">
                       <ColorPicker
+                        popoverAlign="start"
```

Comment:
It was needed because when it's shown in Dialog, a certain portion from left was cut-off